src/components/dialog/dialog.js

```diff
@@ -50,19 +47,19 @@ class DialogElement extends Component {
         theme={theme}
         type='fullscreen' className='modal-dialog'>
         <div className={styles.dialog}>
-          { this.props.dialog.title ?
+          {/* { this.props.dialog.title ?
             <AppBar title={this.props.dialog.title} flat={true}
               className={styles[this.props.dialog.type]}>
               <Navigation type='horizontal'>
-                <IconButton className={`${styles['x-button']} x-button`} onClick={this.goBack.bind(this)} icon='close'/>
+                <IconButton className={`${styles['x-button']} x-button`}
+                  onClick={this.goBack.bind(this)} icon='close'/>
               </Navigation>
             </AppBar> :
-            null }
+            null } */}
           <div className={`modal-dialog-body ${theme.innerBody}`}>
             {this.props.dialog.childComponent ?
               <this.props.dialog.childComponent
                 {...(this.props.dialog.childComponentProps || {})}
-                closeDialog={this.goBack.bind(this)}
               /> :
```

Comment:
I think if we comment the if, then the else or ```: null``` should be removed?

Reply:
It is needed because in some cases it will not render properly